kura/examples/org.eclipse.kura.example.eddystone.advertiser/META-INF/MANIFEST.MF

```diff
@@ -7,8 +7,8 @@ Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Service-Component: OSGI-INF/*.xml
 Import-Package: org.eclipse.kura;version="[1.4,2.0)",
  org.eclipse.kura.ble.eddystone;version="[1.0,2.0)",
- org.eclipse.kura.bluetooth.le;version="[2.0,3.0)",
- org.eclipse.kura.bluetooth.le.beacon;version="[2.0,3.0)",
+ org.eclipse.kura.bluetooth.le;version="[1.0,2.0)",
```

Comment:
Do the same changes need to be applied here?

Reply:
I'm so sure about that... The original commit for the BT doesn't touch these example. They are compatible also with the old version.